containers/Cpp/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,43 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/debian
+{
+	"name": "Azure IoT Edge C/C++",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick an Debian version: bullseye, buster, stretch
+		"args": { "VARIANT": "buster" }
+	},
+	"runArgs": [
+        "--network",
+        "host"
+    ],
+    "mounts": [
+        // Keep command history 
+        "source=ostf-bashhistory,target=/commandhistory,type=volume",
+        // Use hosts docker socket
+        "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind",
+    ],
+
+	// Set *default* container specific settings.json values on container create.
+	"settings": { "terminal.integrated.defaultProfile.linux": "/bin/bash", },
+
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"ms-vscode.azure-account",
+		"ms-vscode.cpptools",
+        "ms-azuretools.vscode-docker",
+        "ms-vscode.azure-account",
+        "vsciot-vscode.azure-iot-toolkit",
+        "vsciot-vscode.azure-iot-tools",
+		"vsciot-vscode.vscode-iot-device-cube"
```

Comment:
we should remove azure-account, iot-toolkit and device-cube as these are dependencies that are brought through iot-tools